src/Uno.UI/UI/Xaml/Controls/ScrollViewer/ScrollViewer.xaml

```diff
@@ -275,8 +275,10 @@
 								Value="{TemplateBinding VerticalOffset}"
 								ViewportSize="{TemplateBinding ViewportHeight}"
 								HorizontalAlignment="Right"
+								skia:SmallChange="16"
 								skia:Width="16"
-								macos:Width="16"/>
+								skia:SmallChange="16"
```

Comment:
duplicated
```suggestion
```

Reply:
About this change, is there a way to not do it in ScrollViewer's template? This feels like the cleanest way to go, but on UWP, SmallChange isn't set in the template, so theoretically, if someone were to copy ScrollViewer's template without this SmallChange property and use it, it would work on Windows but not on the other platforms. This is obviously an extreme edge case, so I'm not sure if we care.

Reply:
it could be moved to scrollviewer's onapplytemplate if needed to,
but i doubt that will be necessary